documentation/messages/user.registration.transactional.md

```diff
@@ -94,10 +94,7 @@ Accessed by connecting to RabbitMQ server and sending message in the following f
   /* Required: user_id or email or mobile. */
   mobile: Integer,
 
-  /* Optional. A string to tag the message with that can be referenced in Mandrill. Example: `drupal_user_register`. */
-  registration_source: String,
-
-  /* Optional. Used to determine the origin of the request as a site in multisite environment. Default: `US`. */
+  /* Optional. Client application identifier. Example: `Phoenix-US`. */
```

Comment:
We'll have to adjust the current config of the Phoenix site to use this value. Should the example use a value that's not in use? I like the format and I think the change would not have impact on any of the systems within Quicksilver.